code/modules/reagents/withdrawal/generic_addictions.dm

```diff
@@ -217,14 +217,15 @@
 		return
 	if(DT_PROB(65, delta_time))
 		return
-	var/obj/item/organ/organ = pick(affected_carbon.internal_organs)
-	if(organ.low_threshold)
-		to_chat(affected_carbon, organ.low_threshold_passed)
-		return
-	else if (organ.high_threshold_passed)
-		to_chat(affected_carbon, organ.high_threshold_passed)
-		return
-	to_chat(affected_carbon, "<span class='warning'>You feel a dull pain in your [organ.name].</span>")
+	if(affected_carbon.stat <= SOFT_CRIT)
```

Comment:
```suggestion
	if(affected_carbon.stat < DEAD)
```
it'd be pretty bullshit to die of organ failure because the message telling you that your organ failed occurred while you were in hardcrit. if you need a fluff reason for this, we can just say that the pain of your organs is influencing your dreams.

Reply:
Nope. If you are unconscious, you dont know what happens to you. Nor are you the one who will be dealing with your issues.